app/views/facility_journals/_problem_transactions.html.haml

```diff
@@ -0,0 +1,68 @@
+.row
+  .span12
+    %p.alert.alert-error= t("facility_journals.new.expired_payment")
+.row
+  .span12
+    = will_paginate(order_details) if order_details.respond_to? :total_pages
+    %table.table.table-striped.table-hover.dense.old-table
+      %thead
+        %tr
+          %th= Order.model_name.human
+          %th= OrderDetail.model_name.human
+          %th.nowrap= OrderDetail.human_attribute_name(@date_range_field)
+          - if @extra_date_column
+            %th.nowrap= OrderDetail.human_attribute_name(@extra_date_column_header || @extra_date_column)
+          - if current_facility.blank? || cross_facility?
+            %th= Facility.model_name.human
+          %th.nowrap= [Account.model_name.human, Account.human_attribute_name("expires_at")].join(" ")
+          %th.order-note= OrderDetail.human_attribute_name("description")
+          %th.nowrap= OrderDetail.human_attribute_name("user")
+          - unless @account
+            %th= Account.model_name.human
+            %th= Account.human_attribute_name("owner")
+          %th.currency= OrderDetail.human_attribute_name("cost")
+          %th= OrderDetail.human_attribute_name("order_status")
+          - if local_assigns[:show_statements]
+            %th= Statement.model_name.human
+      %tbody
+        - order_details.each do |order_detail|
+          %tr{ class: row_class(order_detail) }
+            - if @order_detail_link
+              %td.nowrap= link_to @order_detail_link[:text], @order_detail_link[:proc].call(order_detail) if @order_detail_link[:display?].call(order_detail)
+            - if backend?
+              %td.nowrap= link_to order_detail.order.id, facility_order_path(order_detail.order.facility, order_detail.order)
+              %td.nowrap= link_to order_detail.id, manage_order_detail_path(order_detail), class: 'manage-order-detail'
+            - else
+              %td.nowrap= link_to order_detail.order.id, order_path(order_detail.order)
+              %td.nowrap= link_to order_detail.id, order_order_detail_path(order_detail.order, order_detail)
+            %td= order_detail.send(:"#{@date_range_field}").try(:strftime, "%m/%d/%Y")
```

Comment:
I know this was copied, but we should probably switch to `public_send`. And can we use `format_usa_date`? It's a null-safe version of `I18n.l(date.to_date, format: :usa)`? It'll help our dates stay more consistent.